transcom/mymove
# Int b 18329 remove na from address Kickback

## B-18329

## Summary

When viewing the customer's address as a TOO, the street address is present and would show as "n/a".
This PR shows the origin location on PPMs origin location.
Should also remove "n/a" and only shows city, state, zip for origin and dest location.

### How to test

1. Log in office as TOO
2. select a PPM move
3. Click "Move task order" tab
4. Confirm "customer's addresses" only shows city, state, zip for origin and dest locations.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


## Screenshots
## Before:
<img width="1709" alt="Screenshot 2024-01-22 at 3 49 40 AM" src="https://github.com/transcom/mymove/assets/146969726/f01acb9f-9e5e-4d15-8b77-75c56919dbbb">

## Updated:
<img width="1663" alt="Screenshot 2024-01-22 at 3 50 12 AM" src="https://github.com/transcom/mymove/assets/146969726/aa76bcde-0417-4ca9-96e8-231c280d3109">

## Kickback Updated:
<img width="1171" alt="Screenshot 2024-02-01 at 8 25 17 AM" src="https://github.com/transcom/mymove/assets/146969726/449da43a-00b0-4448-a159-d799fb3fad10">
